src/Core/Content/Media/File/FileFetcher.php

```diff
@@ -142,15 +142,8 @@ private function getUrlFromRequest(Request $request): string
      */
     private function openSourceFromUrl(string $url)
     {
-        $streamContext = stream_context_create([
```

Comment:
We can't remove this by default. We added this long time ago because of [SSRF](https://owasp.org/www-community/attacks/Server_Side_Request_Forgery).  An attacker could create a page which redirects to internal websites and use this feature to request them. 
That's why we disabled here redirects and added an check that the IP resolved cannot be an private one. 

It's fine for me if we couple this to `shopware.media.enable_url_validation`, so when disabled this will be allowed

Reply:
@shyim Okay, can you add this config check to the method openSourceFromUrl?

Reply:
@shyim I have added the check for url validation. Is that change okay?